specs/lightclient/validator.md

```diff
@@ -0,0 +1,198 @@
+
+A validator should plan for future sync committee assignments by noting which sync committee periods they are selected for participation. Specifically, a validator should:
+* Upon (re)syncing the chain, check for assignments in the current and next sync committee periods.
+* If the validator is in the current sync committee period, they can perform the responsibilities below for sync committee rewards.
+* If the validator is in the next sync committee period, they should wait until the next `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` boundary and then perform the responsibilities throghout that period.
+
+*NOTE*: (TODO) trace full range of lookahead...
+
+## Beacon chain responsibilities
+
+A validator maintains the responsibilities given in the Phase 0 document. Block proposals are modified to incorporate the sync committee signatures and detailed below. Validators have a new responsibility during assigned sync committee periods to sign over the span of the chain covered by their sync committee period. These signatures are aggregated and routed to the proposer over gossip for inclusion into a beacon block. Assignments to a particular sync committee will likely be infrequent; however, an action every slot will be required when in the current sync committee.
+
+### Block proposal
+
+Refer to the phase 0 document for the majority of the [block proposal responsibility](../phase0/validator.md#block-proposal). The validator should follow those instructions to prepare a `SignedBeaconBlock` for inclusion into the chain. All changes are additive to phase 0 and noted below.
+
+#### Preparing a `BeaconBlock`
+
+No change to [Preparing for a `BeaconBlock`](../phase0/validator.md#preparing-for-a-beaconblock).
+
+#### Constructing the `BeaconBlockBody`
+
+Each section of [Constructing the `BeaconBlockBody`](../phase0/validator.md#constructing-the-beaconblockbody) should be followed. After constructing the `BeaconBlockBody` as per that section, the proposer gains an additional task to include the sync committee signatures:
+
+##### Sync committee
+
+The proposer will receive a number of `SyncCommitteeContribution`s from validators in the sync committee who are able to broadcast their signature. The proposer will collect all of these contributions for aggregation when preparing a block. `block.body.sync_committee_bits` is a `Bitvector` where the `i`th bit is `True` if the corresponding validator in the sync committee has produced a valid signature. `block.body.sync_committee_signature` is the aggregate BLS signature combining all the valid contributions. Given a collection of `contributions` and the `BeaconBlock` under construction, the proposer should process them as follows:
```

Comment:
If we remove `SyncCommitteeIndex` from the unaggregated message on the wire, the proposer would need to call `indices_in_sync_committee` and do any (maybe duplicate) inclusions for message they have received

Reply:
this work has been moved to the aggregator in the current design